CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+
+
+[2018-07-09]
+--------------------
+#### Added
+- `orx:max-pixels` bind attribute to transform uploaded images.
+- Submission chapter.
+- `orx:auto-send` submission element attribute.
+- `orx:auto-delete` submission element attribute.
+- `<odk:rank>` form control.
+- [`digest`](https://opendatakit.github.io/xforms-spec/#fn:digest) function.
+
+#### Changed
+- Corrected argument order in [`jr:choice-name`](https://opendatakit.github.io/xforms-spec/#fn:jr:choice-name) function.
+- Corrected Psuedo-number generator used in [`randomize`](https://opendatakit.github.io/xforms-spec/#fn:randomize) function.
```

Comment:
psuedo -> pseudo

Reply:
Ah yes, that's my thing. I never know. :(

Reply:
Yes, oh my. This is turning out to be the hardest PR to get approved! :) 👍

Reply:
Oh, it was merged anyway. Woohoo!

Reply:
Heh. We can keep that between us and see if anyone notices.